File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int n) {
        int t=n;
        if(n<0){
            return false;
        }
        int ans=0;
        while(n>0){
            int rem=n%10;
            n=n/10;
            ans=ans*10+rem;
        }
        if(ans!=t){
            return false;
        }
        return true;
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        r=False
        y=x
        if(x<0):
            r=False
        ans=0
        while(x>0):
            rem=x%10
            ans=ans*10+rem
            x=x//10
        if(y!=ans):
            r=False
        else:
            r=True
        return r

